<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Количество ферзей</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="n_edit"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Минимальная температура</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="t_min_edit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Максимальная температура</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="t_max_edit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>α</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="alpha_edit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Число итераций при одной температуре</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="iterations_edit"/>
        </item>
        <item>
         <widget class="QPushButton" name="save_button">
          <property name="text">
           <string>Применить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="show_button_start">
          <property name="text">
           <string>Показать начальное решение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="show_button_best">
          <property name="text">
           <string>Показать лучшее найденное решение</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="MplWidget" name="MplWidget_bs" native="true"/>
        </item>
        <item>
         <widget class="MplWidget" name="MplWidget_temp" native="true"/>
        </item>
        <item>
         <widget class="MplWidget" name="MplWidget_en" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logs"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">MplWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
